CICD/ExtensionMethods.cs

```diff
@@ -899,13 +920,20 @@ public static bool IsReleaseToDoIssue(this Issue issue, ReleaseType releaseType)
             _ => throw new ArgumentOutOfRangeException(nameof(releaseType), releaseType, null),
         };
 
-        var isIssue = issue.PullRequest is null;
+        var isIssue = issue.IsPullRequest();
         var validTitle = issue.Title == releaseLabelOrTitle;
         var validLabelType = issue.Labels.Any(l => l.Name == releaseLabelOrTitle);
 
         return isIssue && validTitle && validLabelType;
     }
 
+    /// <summary>
+    /// Returns a value indicating if a GitHub issue is a release pull request.
```

Comment:
Refactor code comments

```suggestion
    /// Returns a value indicating whether or not a GitHub issue is a release pull request.
```